pkg/config/static/static_config.go

```diff
@@ -280,6 +293,42 @@ func (c *Configuration) initACMEProvider() {
 	legolog.Logger = stdlog.New(log.WithoutContext().WriterLevel(logrus.DebugLevel), "legolog: ", 0)
 }
 
+func (c *Configuration) initHubProvider() error {
+	// Hub provider is an experimental feature. Require the experimental flag to be enabled before continuing.
+	if c.Experimental == nil || !c.Experimental.Hub {
+		return errors.New("experimental flag not set")
+	}
+
+	if c.Hub.TLS == nil && !c.Hub.Insecure {
+		return errors.New("no TLS configuration defined for the Hub provider; either set one or use the --hub.insecure flag for quick testing")
+	}
+
+	if c.Hub.Insecure && c.Hub.TLS != nil {
+		return errors.New("TLS configuration defined and --hub.insecure activated; either set one or use the --hub.insecure flag for quick testing")
```

Comment:
```suggestion
	if c.Hub.Insecure && c.Hub.TLS != nil {
		return errors.New("TLS configuration defined and --hub.insecure activated; either remove the TLS configuration or remove the --hub.insecure flag")
```

Reply:
Should TLS configuration take precedence over `hub.insecure` ?
In this case, if a TLS configuration is set, ignore the `hub.insecure` configuration and/or warn the user if the `hub.insecure` is set.